Automattic/jetpack
# Jetpack Firewall Settings: Redirect users with Protect installed to the standalone plugin's settings page

Resolves https://github.com/Automattic/jetpack-scan-team/issues/1308

## Proposed changes:

* When the Protect plugin is detected on a site, show a prompt to use the Protect firewall settings dashboard instead of the Jetpack settings card.

### Other information:


## Jetpack product discussion



peb6dq-2r8-p2#comment-1717

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Spin up a JN site running this branch with Jetpack.
* Validate the firewall settings continue to display and function as expected.
* Install Jetpack Protect.
* Validate the redirect prompt is shown instead of the firewall settings in Jetpack.

## Screenshots

<img width="1078" alt="Screenshot 2024-07-31 at 1 58 15 PM" src="https://github.com/user-attachments/assets/eb0e4eab-4f91-4054-84c0-69c93ff2d8b7">
